README.md

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you made all your commit message capitalized to make your pull request professional 👍